cmd/root.go

```diff
@@ -0,0 +1,91 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/rs/zerolog/log"
+
+	utils "github.com/fabric8-analytics/cli-tools/utils"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+var debug bool
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "crda",
+	Short: "Cli tool to interact with CRDA Platform.",
+	Long:  `Cli tool to interact with CRDA Platform. Perfoms synk token validation.`,
+	Args:  cobra.ExactValidArgs(1),
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $ROOT/.crda/config.yaml)")
+	rootCmd.PersistentFlags().BoolVarP(&debug, "debug", "d", false, "activate panic")
+	rootCmd.PersistentFlags().String("host", "https://f8a-analytics-preview-2445582058137.production.gw.apicast.io:443", "Host Server, if set, host from config file will be ignored.")
+	rootCmd.PersistentFlags().String("snyk-token", "", "Snyk token, if not set, Freemium account will be created.")
+	rootCmd.PersistentFlags().String("auth-token", "3e42fa66f65124e6b1266a23431e3d08", "3 Scale Token, Token for server authentication.")
```

Comment:
Default values must be represented as string constants.

Reply:
Ack.